model/change_point_detection.go

```diff
@@ -389,14 +388,44 @@ func GetTotalPagesForChangePointsGroupedByVersion(ctx context.Context, env cedar
 		}
 		return int(math.Ceil(float64(res.Count) / float64(pageSize))), nil
 	}
-	return 0, errors.New("Not able to get count of total changepoints matching query")
+	return 0, nil
```

Comment:
If you don't get any results, I don't think its an error. There are just no results.